src/main/java/com/synopsys/integration/detect/lifecycle/run/step/IntelligentModeStepRunner.java

```diff
@@ -241,7 +253,8 @@ public CodeLocationResults calculateCodeLocations(CodeLocationAccumulator codeLo
     private boolean shouldPublishBomLinkForTool(DetectToolFilter detectToolFilter) {
         return detectToolFilter.shouldInclude(DetectTool.SIGNATURE_SCAN) ||
             detectToolFilter.shouldInclude(DetectTool.CONTAINER_SCAN) ||
-            detectToolFilter.shouldInclude(DetectTool.BINARY_SCAN);
+            detectToolFilter.shouldInclude(DetectTool.BINARY_SCAN) ||
+            detectToolFilter.shouldInclude(DetectTool.RL_SCAN);
```

Comment:
I noticed you mentioned there doesn't seem to be a BOM for RL scan? If so, this might not be needed then as this adds the BOM link to the CLI output at the end.

Reply:
Yeah some of this PR is definitely a work in progress. The plan is to wait and display something to the user, so I'll leave this for now, maybe add a TODO to pull it if we aren't showing a BOM.